pygfx/renderers/wgpu/engine/mipmapsutil.py

```diff
@@ -144,17 +149,15 @@ def get_bind_group(
                 dimension="2d",
                 base_array_layer=base_array_layer,
             )
-        dst_view = texture._wgpu_object.create_view(
-            base_mip_level=mip_level,
-            mip_level_count=1,
-            dimension="2d",
-            base_array_layer=base_array_layer,
-        )
+        global mipmap_sampler
+        if mipmap_sampler is None:
+            mipmap_sampler = device.create_sampler(min_filter="linear")
```

Comment:
are you sure it's okay to have a global cache variable? what if this function is invoked for a different `device`?

Reply:
Uh, it will result in an error. However, we have seemingly assumed that only one device instance can exist during program execution (not sure if it's possible to allocate multiple different device instances). Also, it appears that our pipeline cache (and other GPU objects) is global too.

Reply:
@almarklein can you comment on this? I'm not sure

Reply:
Yes, in pygfx we have one single device.